src/MAUI/Maui.Samples/Views/SamplePage.xaml.cs

```diff
@@ -338,19 +327,132 @@ protected override bool OnBackButtonPressed()
+            ToolbarItems.Add(sampleToolbarItem);
+
+            var infoToolbarItem = new ToolbarItem
+            {
+                IconImageSource = "information.png",
+                Text = "About"
+            };
+            infoToolbarItem.Clicked += InfoToolbarItem_Clicked;
+            ToolbarItems.Add(infoToolbarItem);
+
+            var sourceCodeToolbarItem = new ToolbarItem
+            {
+                IconImageSource = "code.png",
+                Text = "Source Code"
+            };
+            sourceCodeToolbarItem.Clicked += SourceCodeToolbarItem_Clicked;
+            ToolbarItems.Add(sourceCodeToolbarItem);
+
+            var gitHubToolbarItem = new ToolbarItem
+            {
+#if WINDOWS
+                IconImageSource = "github.png", // Icon too large on Mac, so force text instead.
+#endif
+                Text = "GitHub"
+            };
+            gitHubToolbarItem.Clicked += GitHubToolbarItem_Clicked;
+            ToolbarItems.Add(gitHubToolbarItem);
+#else
+            var verticalHandle = new ToolbarItem
+            {
+                IconImageSource = "verticalhandle.png"
+            };
+            verticalHandle.Clicked += VerticalHandle_Clicked;
+            ToolbarItems.Add(verticalHandle);
+#endif
+        }
+
+#if WINDOWS
+        private ToolbarItem PrepareScreenshotTool()
+        {
+            var screenshotToolbarItem = new ToolbarItem()
+            {
+                IconImageSource = "camera.png",
+                Text = "Screenshot"
+            };
+            screenshotToolbarItem.Clicked += ScreenshotButton_Clicked;
+
+            SampleContentPage.WidthRequest = ScreenshotManager.ScreenshotSettings.Width.HasValue ? ScreenshotManager.ScreenshotSettings.Width.Value : double.NaN;
+            SampleContentPage.HeightRequest = ScreenshotManager.ScreenshotSettings.Height.HasValue ? ScreenshotManager.ScreenshotSettings.Height.Value : double.NaN;
+            
+            return screenshotToolbarItem;
+        }
+#endif
+
+        private async void VerticalHandle_Clicked(object sender, EventArgs e)
+        {
+            await DisplayActionSheet("Select a view", "Cancel", null, new string[] { "Live Sample", "About", "Source Code", "View on GitHub" }).ContinueWith((result) =>
+            {
+                if (result.Result != "Cancel")
+                {
+                    switch (result.Result)
+                    {
+                        case "Live Sample":
+                            OpenLiveSample();
+                            break;
+
+                        case "About":
+                            OpenDetailsPage();
+                            break;
+
+                        case "Source Code":
+                            OpenSourceCodePage();
+                            break;
+
+                        case "View on GitHub":
+                            _ = OpenGitHub();
+                            break;
```

Comment:
I see these string literals are repeated 3 times in the code.  I'd put these strings into constants, to avoid any sneaky typos / rogue whitespace / future edits from breaking things.  Something very basic like `const string ViewOnGitHub = "View on GitHub";` then using ViewOnGitHub in place of the string literals.